DatadogSessionReplay/SRSnapshotTests/SRHost/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>IsRunningOnCI</key>
+	<string>$(IS_CI)</string>
```

Comment:
/question do we have to configure this or is it set by bitrise?

Reply:
This is already set on CI:
https://github.com/DataDog/dd-sdk-ios/blob/670006d6c3f41ad2652c6f154fd4fda7d04a04a8/Makefile#L38

We use it in other projects through the same approach: by linking `xcconfig` and exporting it to `Info.plist`.